test/rbenv.bats

```diff
@@ -149,39 +149,63 @@ OUT
 
 @test "not enough arguments for rbenv-install" {
   stub_ruby_build
+  stub rbenv-help 'install : echo "Usage: rbenv install"'
```

Comment:
> Rather than merge the two assertions into the same tests, I think I'd rather create an explicit test for 2.

That works. Then keep the `stub rbenv-help` lines, but they don't have to output anything anymore. They can simply be a no-op, and the `unstub rbenv-help` line will verify that rbenv-help was called with the argument it was decleared with (either "install" or "uninstall"). Then remove the "Usage:" output assertions from all tests that don't explicitly test the help text.